llvm/test/Transforms/SampleProfile/pseudo-probe-slotindex.ll

```diff
@@ -1,5 +1,5 @@
-; REQUIRES: x86_64-linux
-; RUN: llc -print-after=slotindexes -stop-after=slotindexes -mtriple=x86_64-- %s -filetype=asm -o %t 2>&1 | FileCheck %s
+; REQUIRES: x86-registered-target
+; RUN: llc -print-after=slotindexes -stop-after=slotindexes %s -filetype=asm -o %t 2>&1 | FileCheck %s
```

Comment:
For llc, removing -mtriple will use the default triple, which may not be x86 (e.g. an AArch64 host might build the x86 target as well).

Reply:
Thanks for clarification.